Sources/Onboarding/OnboardingVPNDetailsView.swift

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
```

Comment:
2022

Reply:
Maybe we should add `IDETemplteMacros` so I will not do this mistake everytime :)

Fix at https://github.com/brave/brave-ios/pull/6355/commits/b5c9b38224db9ddb676f6eea2068a7a202dd541c

Reply:
it broke with our move to the SPM architecture, what Kyle told me

Reply:
Yeah Xcode doesn't seem to respect the files created inside the package _unless_ you open the Package.swift directly in its own workspace. Our `IDETemplateMacros` is correct in both the main Xcode workspace and the Package.swift one but unfortunately it doesn't get used in most cases for either :(

Reply:
I recommend making a code snippet with the header in Xcode and just give it a short completion (mine is `mpl2`). Its not as good as Xcode automatically doing it but its still fast enough

Reply:
Sounds good, thanks guys